firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -23,6 +23,13 @@ enum EighthStepsScale
   NEG_EIGHTH_STEPS_PER_EIGHTH_STEP = -1
 };
 
+// Sets position extremes to be used as alignment codes
+enum positionExtremes
+{
+  POS_EXTREME = 1,
+  NEG_EXTREME = 0
+};
+
```

Comment:
I don't think we need an enum here since there are only two cases, moving to max, or moving to zero. We can update the function header to have a bool as the "alignment code". I'll add a comment there explaining, but you can delete this.

I realize many of my comments could have been avoided if I had looked at the issue @LSaber36 made in advance, so sorry for that.